server/parser/src/main/java/org/eclipse/lsp/cobol/core/hw/CobolLexer.java

```diff
@@ -28,8 +29,8 @@
  */
 public class CobolLexer {
   private final String source;
-  private Position position = new Position();
-  private Position lastTokenStartPosition = new Position();
+  @Getter private Position position = new Position();
```

Comment:
Not sure , if about this change.
I will rework to get token locality for diagnostics